### DISCUSSION TITLE
NeMo Toolkit benchmarks

### DISCUSSION THREAD

**Discussion by itzsimpl** [NVIDIA-NeMo/NeMo#8033] [General]
Are there any NeMo toolkit benchmarks? More specifically, I'm searching for reference values of the expected performance in training ASR models on a single DGX-A100 320GB, or DGX-H100 640GB system. A repeatable recipe would be perfect.

**Comment by titu1994** [comment]
We dont perform benchmarking for the NeMo toolkit. There are numbers for NeMo framework available but that is for LLMs training only.
